bucket/DIE.json

```
@@ -0,0 +1,47 @@
+{
+    "description": "Detect It Easy, or abbreviated \"DIE\" is a program for determining types of files.",
+    "homepage": "https://ntinfo.biz/index.html",
+    "license": "MIT",
+    "version": "2.05",
+    "url": "https://github.com/horsicq/DIE-engine/releases/download/2.05/die_win32_portable_2.05.zip",
+    "hash": "c597381d73a7ce6ad1acc216c5ddd9010cde56c830063eaeee9411974f60963e",
+    "extract_dir": "die_win32_portable/base",
```

Why did you extract base??

```suggestion
    "extract_dir": "die_win32_portable\\base",
```

---

I guess it's not necessary. The actual application is in base, and there are just 2 shims next to it, which I think just clutters things, especially when installed via scoop.

---

Then why there are executables outside of this folder?

---

Sorry, I'm not sure what you are asking. The zip file contents look like this:
```
die_win32_portable
├── die.exe
├── diel.exe
└── base
    ├── diec.exe
    ├── die.exe
    ├── diel.exe
    └── ...
```
The actual application is in the `base` subdirectory and there are just 2 shims pointing to it. That seems unnecessary, so I just had it extract the `base` directory.

Does that make sense? Or am I misunderstanding something?

---

The executables at `die_win32_portable\die.exe` and `die_win32_portable\diel.exe` are just shims pointing to their base counterparts.

---

> The executables at die_win32_portable\die.exe and die_win32_portable\diel.exe are just shims pointing to their base counterparts.

Source of this claim?

---

Personal experience and testing. I've been installing it this way for a while, and if you attempt to run one without base, it will give you an error telling you it's missing. 

I also just opened it up in x64dbg real quick and confirmed in the disassembly. I could go look through the actual source code if you really want me to, or if it bothers you, you can just leave base and the shims in.

---

https://github.com/horsicq/DIE-engine/blob/master/die_source/winloader_source/winloader_die.cpp

---

> > The executables at die_win32_portable\die.exe and die_win32_portable\diel.exe are just shims pointing to their base counterparts.
> 
> Source of this claim?

I can confirm based on my testing.